bc_obps/registration/migrations/0031_activity_model.py

```diff
@@ -0,0 +1,176 @@
+                'db_table': 'erc_history"."activity_history',
+                'ordering': ('-history_date', '-history_id'),
+                'get_latest_by': ('history_date', 'history_id'),
+            },
+            bases=(simple_history.models.HistoricalChanges, models.Model),
+        ),
+        migrations.RemoveField(
+            model_name='operation',
+            name='reporting_activities',
+        ),
+        migrations.AddField(
+            model_name='operation',
+            name='activities',
+            field=models.ManyToManyField(blank=True, related_name='%(class)ss', to='registration.activity'),
+        ),
+        migrations.CreateModel(
+            name='HistoricalOperation_activities',
+            fields=[
+                ('id', models.BigIntegerField(auto_created=True, blank=True, db_index=True, verbose_name='ID')),
+                ('m2m_history_id', models.AutoField(primary_key=True, serialize=False)),
+                (
+                    'activity',
+                    models.ForeignKey(
+                        blank=True,
+                        db_constraint=False,
+                        db_tablespace='',
+                        null=True,
+                        on_delete=django.db.models.deletion.DO_NOTHING,
+                        related_name='+',
+                        to='registration.activity',
+                    ),
+                ),
+                (
+                    'history',
+                    models.ForeignKey(
+                        db_constraint=False,
+                        on_delete=django.db.models.deletion.DO_NOTHING,
+                        to='registration.historicaloperation',
+                    ),
+                ),
+                (
+                    'operation',
+                    models.ForeignKey(
+                        blank=True,
+                        db_constraint=False,
+                        db_tablespace='',
+                        null=True,
+                        on_delete=django.db.models.deletion.DO_NOTHING,
+                        related_name='+',
+                        to='registration.operation',
+                    ),
+                ),
+            ],
+            options={
+                'verbose_name': 'HistoricalOperation_activities',
+            },
+            bases=(simple_history.models.HistoricalChanges, models.Model),
+        ),
+        migrations.DeleteModel(
+            name='HistoricalOperation_reporting_activities',
+        ),
+        migrations.RunSQL("""
+            INSERT INTO erc.activity (
+                name,
+                applicable_to,
+                slug,
+                weight
+            )
+            SELECT
+                name,
+                applicable_to,
+                slug,
+                weight
+            FROM
+                erc.reporting_activity;
+        """, reverse_sql="""
+            truncate erc.activity restart identity cascade;
+        """)
```

Comment:
No big deal here, but I suggest using Django's ORM instead of writing raw SQL to maintain consistency across the codebase.